aider.el

```diff
@@ -191,6 +203,17 @@ COMMAND should be a string representing the command to send."
       ;; Use the shared helper function to send the command
       (aider--send-command command))))
 
+;; Function to send "/read <current buffer file full path>" to corresponding aider buffer
+(defun aider-read-current-file ()
+  "Send the command \"/read <current buffer file full path>\" to the corresponding aider comint buffer."
+  (interactive)
+  ;; Ensure the current buffer is associated with a file
+  (if (not buffer-file-name)
+      (message "Current buffer is not associated with a file.")
+    (let ((command (format "/read %s" (expand-file-name buffer-file-name))))
```

Comment:
Thanks for adding this function. I didn't see the /read command in aider usage page: https://aider.chat/docs/usage/commands.html, would you mind point me to the document for it?

Reply:
I didn't realize it was undocumented but it mentioned a few times in the docs or faqs

 - https://aider.chat/docs/faq.html#can-i-use-aider-with-multiple-git-repos-at-once
 - https://aider.chat/docs/faq.html#can-i-use-aider-with-multiple-git-repos-at-once